docs/reference/overview_of_strategies.rst

```diff
@@ -250,7 +250,91 @@ This strategy plays randomly (disregarding the history of play).
 Axelrod's second tournament
 ---------------------------
 
-Work in progress.
+The code for Axelrod's second touranment was originally published by the
+`University of Michigan Center for the Study of Complex Systems <http://lsa.umich.edu/cscs/>`_
+and is now available from
+`Robert Axelrod's personal website <http://www-personal.umich.edu/~axe/research/Software/CC/CC2.html>`_
```

Comment:
Can we copy the actual permissions "You are free to download these materials and use them without restriction" since that page may not last forever? Or am I being paranoid?